src/maks.kulykov/controller/Controller.java

```diff
@@ -51,27 +53,106 @@ public void run() {
         boolean isExit = false;
 
         while (!isExit) {
+            String[] commandData = new String[2];
+            if (command.startsWith("find|")) {
+                commandData = command.split("\\|");
+                command = commandData[0];
+            }
+
             switch (command) {
                 case "help" -> {
                     view.write("Command list:");
                     view.write("list - show list of all tables");
+                    view.write("find|tableName - show table data");
                     view.write("help - show list of all command");
                     view.write("exit - exit program");
                     command = view.read();
                 }
                 case "list" -> {
-                    view.write(manager.getTablesList());
+                    printTablesList(manager.getTablesList());
                     command = view.read();
                 }
                 case "exit" -> {
                     view.write("See you!");
                     isExit = true;
                 }
+                case "find" -> {
+                    ArrayList<String> columnNames = manager.getTableHeaders(commandData[1]);
+                    int columns = columnNames.size();
+                    printLine(columns);
+                    printHeader(columnNames);
+                    printLine(columns);
+
+                    Set<HashMap<String, String>> tableData = manager.getTableData(commandData[1]);
```

Comment:
Use interfaces for HashMap -> Map, for ArraryList -> List for variables and parameters.

Reply:
+